legal-api/tests/unit/services/test_authorization.py

```diff
@@ -542,13 +542,13 @@ def mock_auth(one, two):  # pylint: disable=unused-argument; mocks of library me
          ['CP', 'BC', 'BEN', 'CC', 'ULC', 'LLC'], 'general', [BASIC_USER], False),
 
         ('user_active_allowed', Business.State.ACTIVE, 'dissolution', 'voluntary',
-         ['CP', 'BC', 'BEN', 'CC', 'ULC'], 'general', [BASIC_USER], True),
+         ['CP', 'BC', 'BEN', 'CC', 'ULC'], 'general', [BASIC_USER], False),
```

Comment:
Same comment as above regarding how voluntary dissolution should return a `True` and that `SP`, `GP` should be added.